cms/search/templates/search/search.html

```diff
@@ -20,34 +25,42 @@ <h2 class="nhsuk-heading-xs"></h2>
     </div>
   </div>
   <div class="nhsuk-grid-row nhsuk-u-margin-top-8">
-    <div class="nhsuk-grid-column-one-third">
-      <h2 class="nhsuk-heading-xs">Filters</h2>
-      {% include 'partials/nhs_components/search_content_type.html' %}
-    </div>
-    {# body#}
-    <div class="nhsuk-grid-column-two-thirds">
-      <div class="search-results-heading">
-        {% include 'partials/nhs_components/select_search_results_by_order.html' %}
+  <div class="nhsuk-grid-column-one-third">
```

Comment:
This looks like an unexpected indentation change which makes the diff bigger than it should be.

Reply:
Yep, the HTML indents are causing the big diff.